cmd/promoted-image-governor/main.go

```diff
@@ -174,6 +205,57 @@ type ImageStreamRef struct {
 	ExcludeTags []string `json:"excludeTags,omitempty"`
 }
 
+// OpenshiftMappingConfig for openshift image mapping files
+type OpenshiftMappingConfig struct {
+	SourceRegistry  string              `json:"source_registry"`
+	TargetRegistry  string              `json:"target_registry"`
+	SourceNamespace string              `json:"source_namespace"`
+	TargetNamespace string              `json:"target_namespace"`
+	Images          map[string][]string `json:"images,omitempty"`
+}
+
+func generateMappings(promotedTags []api.ImageStreamTagReference, mappingConfig *OpenshiftMappingConfig, imageStreamRefs []ImageStreamRef) map[string]map[string][]string {
```

Comment:
I think it might be cool to have some docs on this to explain the intent behind what it's doing. It's probably just because I'm new, but without going to look at the attached card and accompanying release PR I struggled a little to figure out what we were doing here.

Reply:
I will do tomorrow.